starter-code/javascript/chronometer.js

```diff
@@ -2,49 +2,57 @@
 function Chronometer() {
   this.currentTime = 0;
   this.intervalId = 0;
+  this.minutes ="";
+  this.seconds ="";
  }
 
 Chronometer.prototype.startClick = function () {
  this.intervalId = setInterval(function(){
+   this.setTime();
```

Comment:
el orden es muy importante - aquí debería estar el this.currentTime++ primero y luego this.setTime(). También, en el test os dice que se está sumando 100 por cada iteración cuando solo debería sumarse 1 - esto se cambia con ponerlo a 1000 milisegundos en vez de 10.